### DISCUSSION TITLE
vbsp.exe a virus?

### DISCUSSION THREAD

**Discussion by John2027** [BEEmod/BEE2-items#4170] [Tech Support / Q&A] [answered]
When I try to use Beemod, Avast says the vbsp.exe is infected with IDP.Generic. Which means that a file is acting strangely, which can be signs of malware.

**Comment by TeamSpen210** [comment, marked as answer]
Yes, because it's pretending to be the original compilers. It's not a virus.

**Comment by FrozenDragon0** [comment]
@Fazbear189 Like @TeamSpen210 said it's not a virus it's the way it acts that give anti-viruses false alarms.

**Comment by MxrcY1** [comment]
the only problem is that i cant delet any of the bee2 files.

edit: btw malwarebytes, avast and AVG says it's not a virus. but if i want to play a soundtrack avast blocks it cuz it's "ransomware".
